clients/guzzle6-adapter.rst

```diff
@@ -1,2 +1,39 @@
-Guzzle6 Adapter
-===============
+Guzzle 6 Adapter
+================
+
+An HTTPlug adapter for the `Guzzle 6 HTTP client`_.
+
+First, create a Guzzle client passing any options you like::
+
+    $config = [
+        // Config params
+    ];
+    $guzzle = new GuzzleHttp\Client($config);
+
+Then create the adapter::
+
+    use Http\Adapter\Guzzle6\Client;
+
+    $adapter = new Client($guzzle);
+
+And use it to send synchronous requests::
+
+    use GuzzleHttp\Psr7\Request;
+
+    $request = new Request('GET', 'http://httpbin.org');
```

Comment:
I wanted to keep things as easy as possible for new users. 

And because guzzle6-adapter requires guzzle which requires guzzle/psr7, I thought it made sense to just use Guzzle’s PSR-7 Request object here. 

Added a reference to the message factory doc chapter.